src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -939,3 +967,56 @@ void AppHost::_SummonWindowRequested(const winrt::Windows::Foundation::IInspecta
     summonArgs.ToggleVisibility(false); // Do not toggle, just make visible.
     _HandleSummon(sender, summonArgs);
 }
+
+void AppHost::_HandleTrayIconPressed()
+{
+    // Currently scoping "minimize to tray" to only
+    // the quake window.
+    if (_logic.IsQuakeWindow())
+    {
+        const Remoting::SummonWindowBehavior summonArgs{};
+        summonArgs.DropdownDuration(200);

```

Comment:
Is this `200` just the number that we use specifically for the quake mode key binding?

Mildly concerned that if we ever decide to change the dropdown duration, we'll forget that this one exists. Maybe `Remoting::SummonWindowBehavior` should have a static function dedicated to building the quake mode `summonArgs`? That way it's consistent throughout?

(if you agree, I'm ok with this being a follow-up, or being tracked elsewhere, just be sure to add a TODO comment and tag this up)

Reply:
I'll stick it in my tray icon / minimize to tray todos